src/main.c

```diff
@@ -107,7 +107,7 @@ HBlankFunc gHBlankIntrs[4] ALIGNED(16) = {};
 u8 gIwramHeap[] = {};
 EWRAM_DATA u8 gEwramHeap[] = {};
 
-Sprite *gUnknown_03004D10[] ALIGNED(16) = {};
+Sprite *gUnknown_03004D10[16] ALIGNED(16) = {};
```

Comment:
FYI the size of these are being declared in the header as that means we only need to declare in one place

Reply:
Oh, that makes sense!
It was just a leftover from when I discovered its type! :)